packages/scheduler-prisma/src/helpers.ts

```diff
@@ -17,39 +17,88 @@ const SIX_MONTHS_IN_MS = 15778476000;
 export const isHealthy = (heartbeat: number, timeout: number) =>
   new Date().getTime() - timeout < heartbeat;
 
+const getValidRule = (recurrence: string, timezone?: string) => {
+  const isICalRule = recurrence.includes('DTSTART');
+  if (isICalRule) return recurrence;
+
+  let derivedTimezone = timezone ?? 'Australia/Sydney';
+  const rule = recurrence
+    .split(';')
+    .filter(b => {
+      const [key, value] = b.split('=');
+      if (key === 'TZID') {
+        derivedTimezone = value;
+      }
+      return key !== 'TZID';
+    })
+    .join(';');
+
+  const timeISO8601 = moment().tz(derivedTimezone).format('YYYYMMDDTHHmmss');
+  return `DTSTART;TZID=${derivedTimezone}:${timeISO8601}\nRRULE:${rule}`;
+};
+
 // interval should be iCal String.
-export const computeNextRunAt = (
```

Comment:
We are deprecating this method by dropping it (client of this api will fail to compile). Could you please update the README document with:
1. What method was dropped and should be replaced by (maybe add why)
2. How is the new one different (e.g. defaults to Sydney time)
3. An example to convert usage of `computeNextRunAt` to `computeNextRun`.